modules/gapi/test/s11n/gapi_sample_pipelines_s11n.cpp

```diff
@@ -806,4 +806,33 @@ TEST(S11N, Pipeline_Render_RGB)
 
     EXPECT_EQ(cv::norm(input,  ref_mat), 0);
 }
+
+TEST(S11N, Pipeline_Const_GScalar)
+{
+    static constexpr auto in_scalar = 10;
+
+    cv::GMat a;
+    cv::GScalar s;
+
+    cv::GComputation computation(GIn(a), GOut(cv::gapi::addC(a, in_scalar)));
+    auto p = cv::gapi::serialize(computation);
+    auto deserialized_computation = cv::gapi::deserialize<cv::GComputation>(p);
+
+    cv::Mat in_mat = cv::Mat::eye(32, 32, CV_8UC1);
+    cv::Mat ref_mat;
+    cv::add(in_mat, in_scalar, ref_mat);
+
+    cv::Mat out_mat;
+    computation.apply(cv::gin(in_mat/*, in_scalar*/), cv::gout(out_mat));
```

Comment:
What is the reason for leaving commented out in_scalar

Reply:
I used it to emphasize  that scalar in no runtime param here. But maybe it's excessive
Shall I remove it?

Reply:
no, that's OK